### ISSUE TITLE
Unable to see my wishlist in the browser

### ISSUE THREAD

**Issue by CTOUT** [IsThereAnyDeal/AugmentedSteam#1954]
I had an issue with seeing my wishlist in the browser.

Message when viewing my wishlist (worked as expected in the app): -

`OOPS, YOU CAN'T VIEW THIS WISHLIST`
`You don't have permission to view this user's wishlist.`

I'm logged in as myself (obviously), I've tried changing my privacy settings to public, and I also now get it in the Steam app after the wishlisted items had been removed.  This subsequently resolved itself in the Steam app and the website after the wishlist was emptied - it might well have been a Steam glitch rather than an Augmented one.

_Originally posted by @CTOUT in https://github.com/IsThereAnyDeal/AugmentedSteam/issues/1952#issuecomment-2204220707_

**Comment by candela97** [comment]
I don't think this has to do with AS, unless it can be reproduced with AS enabled/disabled.

**Comment by CTOUT** [comment]
I can't reproduce it now - so likely a Steam-side issue.
